app/views/pages/api-reference/liquid/variables.liquid

```diff
@@ -1,5 +1,6 @@
 ---
 converter: markdown
+enable_profiler: true
```

Comment:
This shouldnt go to production @diana-lakatos

Reply:
@Slashek Please advise, thank you.

Reply:
Fixed, thank you!